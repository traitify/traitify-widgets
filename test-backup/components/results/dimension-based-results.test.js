import Component from "components/results/dimension-based-results";
import ComponentHandler from "support/component-handler";

jest.mock("components/results/dimension-based-results/radar", () => (() => (<div className="mock">Radar</div>)));
jest.mock("components/results/dimension-based-results/personality-types", () => (() => (<div className="mock">Personality Types</div>)));
jest.mock("components/results/dimension-based-results/personality-heading", () => (() => (<div className="mock">Personality Heading</div>)));
jest.mock("components/results/dimension-based-results/personality-details", () => (() => (<div className="mock">Personality Details</div>)));
jest.mock("components/results/dimension-based-results/dimensions", () => (() => (<div className="mock">Dimensions</div>)));
jest.mock("components/results/dimension-based-results/personality-traits", () => (() => (<div className="mock">Personality Traits</div>)));
jest.mock("lib/with-traitify");

describe("DimensionBasedResults", () => {
  it("renders results", () => {
    const component = new ComponentHandler(<Component />);

    expect(component.tree).toMatchSnapshot();
  });
});
