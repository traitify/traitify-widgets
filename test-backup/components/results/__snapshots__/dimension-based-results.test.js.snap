// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DimensionBasedResults renders results 1`] = `
<section>
  <div
    className="mock"
  >
    Radar
  </div>
  <div
    className="mock"
  >
    Personality Types
  </div>
  <div
    className="mock"
  >
    Personality Heading
  </div>
  <div
    className="mock"
  >
    Personality Details
  </div>
  <div
    className="mock"
  >
    Dimensions
  </div>
  <div
    className="mock"
  >
    Personality Traits
  </div>
</section>
`;
